python/grass/pygrass/vector/table.py

```diff
@@ -859,10 +859,10 @@ def connection(self):
                 return psycopg2.connect(db)
             except ImportError:
                 er = "You need to install psycopg2 to connect with this table."
-                raise ImportError(er)
+                raise ImportError(er) from None
         else:
             str_err = "Driver is not supported yet, pleas use: sqlite or pg"
-            raise TypeError(str_err)
+            raise TypeError(str_err) from None
```

Comment:
Wouldn't the `except ImportError: raise ImportError` cases be best handled by `add_note...raise`? I would say yes, except that we need to wait for min Python on main version to be raised to 3.11 ([PEP](https://peps.python.org/pep-0678/)).

Reply:
It's always possible to use the feature if it is available and let it go if not supported. Or do a fallback of some sort, that adds the text to the message instead.